lib/shopify_transporter/exporters/magento/customer_exporter.rb

```diff
@@ -15,12 +15,10 @@ def key
 
         def export
           base_customers.each do |customer|
-            begin
-              yield with_attributes(customer)
-            rescue Savon::Error => e
-              print_customer_details_error(customer, e)
-              yield customer
-            end
+            yield with_attributes(customer)
```

Comment:
We ran into this with @fahia's pr as well - this rubocop suggestion is actually broken/is wrong because the code doesn't work anymore.

I suggest seeing if we can skip this check with @fahia.

Reply:
I see. Thanks for pointing it out! I'll revert the previous commit.